config/envs/development.py

```diff
@@ -12,3 +12,18 @@ class DevelopmentConfig(DefaultConfig):
 
     FSD_LOGGING_LEVEL = logging.DEBUG
     USE_LOCAL_DATA = strtobool(getenv("USE_LOCAL_DATA", "True"))
+
+    COOKIE_DOMAIN = getenv("COOKIE_DOMAIN", ".localhost.localhost")
```

Comment:
Is `.localhost.localhost` the correct value for local development? Or is it OK for this to be None?

Reply:
Actually unused here though I think! Removed